README.md

```diff
@@ -4,54 +4,129 @@ A library for creating and managing feeds using Hyperdrive and Hyperswarm.
 
 ## Usage
 
-Initiate the library.
-
 ```js
-const feeds = new Feeds();
-const feedID = feeds.randomID();
-await feeds.update(feedID, 'balance', 1000);
+const { Client, Relay } = require('@synonymdev/web-relay')
+const path = require('path')
+
+const { Feed, Reader } = require('@synonymdev/feeds')
+
+const relay = new Relay(path.join(__dirname, '/storage/relay'));
+
+(async () => {
+  const relayAddress = await relay.listen()
+
+  const client = new Client({ storage: path.join(__dirname, '/storage/writer'), relay: relayAddress })
+
+  const icon = Buffer.from('icon-data')
+
+  const config = {
+    name: 'price-feed',
+    description: 'a price feed',
+    icons: {
+      32: 'icon.png'
+    },
+    type: 'price-feed',
+    version: '1.0.0',
+    fields: [
+      {
+        name: 'latest',
+        description: 'Bitcoin / US Dollar price history',
+        main: '/feed/BTCUSD-last'
+      }
+    ]
+  }
+
+  const feed = new Feed(client, config, { icon })
+  // Wait for config file to be saved
+  await feed.ready()
+
+  // may need to wait more for a the relay to get the files from the client in production.
+  // await new Promise(resolve => settimeout(resolve, 200))
+
+  feed.put('BTCUSD-last', 1000000)
+
+  {
+    const client = new Client({ storage: path.join(__dirname, '/storage/reader') })
+    const reader = new Reader(client, feed.url)
+
+    console.log('Config:', await reader.getConfig())
+    console.log('BTCUSD-last', await reader.getField('BTCUSD-last'))
+  }
+
+  relay.close()
+})()
 ```
 
 ## API
 
-#### `const feeds = new Feeds(storage, [opts])`
+#### `const feed = new Feed(client, config, [opts])`
+
+Create a Feed instance.
 
-Create a Feeds instance.
+`client` is a [web-relay](https://github.com/slashtags/web-relay) client
```

Comment:
This is _fine_ for now since we only have 1 protocol to support. 
We will need to come back and decouple the requirement of using the specific web-relay and instead make the client be something like Slashtags Protocol class, which could be web-relay or something else.